lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -42,6 +51,12 @@ $proxy_url = $module.Params.proxy_url
 $proxy_username = $module.Params.proxy_username
 $proxy_password = $module.Params.proxy_password
 $force = $module.Params.force
+$checksum = $module.Params.checksum
+$checksum_algorithm = $module.Params.checksum_algorithm
+$checksum_url = $module.Params.checksum_url
+
+$module.Diff.before = @{}
```

Comment:
I don't think the diff is useful right now. A change for the module means the file was changed. If we do want to support diff in the future we should look at showing the content diff. I think for now it's best to leave this out for a separate PR so we can focus on the checksum work right now.

Reply:
I guess we need it for local tests.
Moreover if you want improve/refactor diff you should create separate PR.
It's not only adding checksum feature - it's full module refactoring in result.
As for me it's enough to see what files was changed (including temp location).
Do you want to see content diffs for files more than 100mb? It will be pain for eyes, didn't it?
Does it make sense?

Reply:
@jborean93 Could you please take a look?

Reply:
I suppose that's true. I would just feel more comfortable if we focus on the checksum features in this PR and then look at the diff addition in a future PR. The changes that have been made already are quite substantial as it is.